sdk/scheduler/src/test/java/com/mesosphere/sdk/config/validate/UserCannotChangeTest.java

```diff
@@ -0,0 +1,80 @@
+package com.mesosphere.sdk.config.validate;
+
+import com.mesosphere.sdk.specification.*;
+import com.mesosphere.sdk.testutils.TestConstants;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+
+import java.util.Arrays;
+import java.util.Optional;
+
+import static org.mockito.Mockito.when;
+
+public class UserCannotChangeTest {
```

Comment:
Moar tests:
1. compare pods not setting any user
1. compare multiple pods
1. compare pod with no user with one with a suer